### DISCUSSION TITLE
process_linux.go:297: applying cgroup configuration for process caused \\\"mkdir /sys/fs/cgroup/net_cls,net_prio/system.slice/docker-ee11c0cf07aac68ff42ea0c421fe1a0d2d255532a98ffff2b86151e88685052a.scope: no space left on device\\\"\": unknown

### DISCUSSION THREAD

**Discussion by LinshanYu** [moby/moby#45011] [Q&A]
help!
There was a bug reported by docker , which was "Linux 61e2d988 5.4.56-200.el7.x86_64 #1 SMP Sat Aug 8 21:08:01 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux" , when i tried to run a container by docker.
os 
Linux 61e2d988 5.4.56-200.el7.x86_64 #1 SMP Sat Aug 8 21:08:01 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux
docker version 
Client: Docker Engine - Community
 Version:           19.03.13
 API version:       1.40
 Go version:        go1.13.15
 Git commit:        4484c46d9d
 Built:             Wed Sep 16 17:03:45 2020
 OS/Arch:           linux/amd64
 Experimental:      false

Server: Docker Engine - Community
 Engine:
  Version:          19.03.13
  API version:      1.40 (minimum version 1.12)
  Go version:       go1.13.15
  Git commit:       4484c46d9d
  Built:            Wed Sep 16 17:02:21 2020
  OS/Arch:          linux/amd64
  Experimental:     false
 containerd:
  Version:          1.3.7
  GitCommit:        8fba4e9a7d01810a393d5d25a3621dc101981175
 runc:
  Version:          1.0.0-rc10
  GitCommit:        dc9208a3303feef5b3839f4323d9beb36df0a9dd
 docker-init:
  Version:          0.18.0
  GitCommit:        fec3683
docker info
Client:
 Debug Mode: false

Server:
 Containers: 6
  Running: 5
  Paused: 0
  Stopped: 1
 Images: 5
 Server Version: 19.03.13
 Storage Driver: overlay2
  Backing Filesystem: xfs
  Supports d_type: true
  Native Overlay Diff: true
 Logging Driver: json-file
 Cgroup Driver: systemd
 Plugins:
  Volume: local
  Network: bridge host ipvlan macvlan null overlay
  Log: awslogs fluentd gcplogs gelf journald json-file local logentries splunk syslog
 Swarm: inactive
 Runtimes: runc
 Default Runtime: runc
 Init Binary: docker-init
 containerd version: 8fba4e9a7d01810a393d5d25a3621dc101981175
 runc version: dc9208a3303feef5b3839f4323d9beb36df0a9dd
 init version: fec3683
 Security Options:
  seccomp
   Profile: default
 Kernel Version: 5.4.56-200.el7.x86_64
 Operating System: CentOS Linux 7 (Core)
 OSType: linux
 Architecture: x86_64
 CPUs: 32
 Total Memory: 94.28GiB
 Name: 61e2d988
 ID: ZUVB:JSBG:DBXA:7XDL:FKNY:WY27:NZ7S:MKW6:6QRP:VMW2:63TO:IVDY
 Docker Root Dir: /var/lib/docker
 Debug Mode: false
 Registry: https://index.docker.io/v1/
 Labels:
 Experimental: false
 Insecure Registries:
  127.0.0.0/8
thk

**Comment by LinshanYu** [comment]
@uggedal 
@evanphx

**Reply by evanphx** [reply]
Don't tag random people, it's very rude.
